airflow/serialization/serialized_objects.py

```diff
@@ -698,6 +708,15 @@ def deserialize(cls, encoded_var: Any, use_pydantic_models=False) -> Any:
             return decode_timezone(var)
         elif type_ == DAT.RELATIVEDELTA:
             return decode_relativedelta(var)
+        elif type_ == DAT.AIRFLOW_EXC_SER:
+            print(f"var={var}")
+            deser = cls.deserialize(var, use_pydantic_models=use_pydantic_models)
+            exc_cls_name = deser["exc_cls_name"]
+            args = deser["args"]
+            kwargs = deser["kwargs"]
+            del deser
+            exc_cls = import_string(f"airflow.exceptions.{exc_cls_name}")
+            return exc_cls(*args, **kwargs)
```

Comment:
One big problem with re-instantiating exceptions is it looses the tracebacks. I’m not sure if it’s possible but may be worthwhile looking into.

Reply:
In this case it doesn’t matter though because these exceptions aren’t errors

Reply:
Yeah. And in this cases those would be "Remote" tracebacks, not really useful locally.